### PR TITLE
config: add suppressions for indetation and javadoc to avoid asking pitest sharing from contributors

### PR DESCRIPTION
**Pull request by romani**

### CODE REVIEW

File: .ci/shippable.sh [hunk 1]
```diff
@@ -78,9 +78,105 @@ pitest-checks-coding)
...
+  "TagParser.java.html:<td class='covered'><pre><span  class='survived'>                while (column &#60; currentLine.length()</span></pre></td></tr>"
+  "WriteTagCheck.java.html:<td class='covered'><pre><span  class='survived'>            log(lineNo, MSG_MISSING_TAG, tag);</span></pre></td></tr>"
+  "WriteTagCheck.java.html:<td class='covered'><pre><span  class='survived'>                    tagCount += 1;</span></pre></td></tr>"
+  );
+  checkPitestReport "${ignoredItems[@]}"
+  ;;
+
+pitest-checkstyle-gui)
+  mvn -e -P$1 clean test org.pitest:pitest-maven:mutationCoverage;
+  # post validation is skipped, we do not test gui throughly
```
Comment (rnveach) [inline .ci/shippable.sh]: will we ask users for this when they modify gui?

Comment (romani) [reply .ci/shippable.sh]: no, we will not enforce pitest over gui code at all.

Comment (rnveach) [reply .ci/shippable.sh]: Why do we even run pitest for gui then?

Comment (romani) [reply .ci/shippable.sh]: good point, disabled in separate(2nd) commit, to not waste time of CI.

**Comment by rnveach** [comment]
@romani This is the 2nd time wercker has failed with this error. Something is up.
> setup environment
no such image

**Comment by romani** [comment]
manually squashed and merged.

**Pull request closed without merging**